.gitignore

```diff
@@ -23,4 +23,4 @@ Version.cpp
 CMakeCache.txt
 CMakeFiles/*
 cmake_install.cmake
-
+.idea/
```

Comment:
This change is not really needed.

Reply:
It's not needed for the project as such, but prevents people from checking in their CLion stuff.